lib/ipaddr.h

```diff
@@ -93,6 +93,13 @@ static inline char *ipaddr2str(const struct ipaddr *ip, char *buf, int size)
 	return buf;
 }
 
+#define VALIDATE_MAPPED_IPV6(A)                                                \
```

Comment:
Can we rename this? It says it does validation but all its really doing is checking if its an ipv4 embedded ipv6 address.

I think naming it `IS_MAPPED_IPV6(A)` would make the other code easier to read.

Reply:
Yes sure, it is renamed now. 
 
Thanks,